<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>648</width>
    <height>750</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Production Line Simulator</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="rootLayout">
    <item>
     <widget class="QScrollArea" name="scrollArea">
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="scrollAreaWidgetContents_2">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>628</width>
         <height>707</height>
        </rect>
       </property>
       <widget class="QWidget" name="">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>630</width>
          <height>28</height>
         </rect>
        </property>
        <layout class="QHBoxLayout" name="toolbarLayout">
         <item>
          <widget class="QPushButton" name="btnStart">
           <property name="text">
            <string>Start</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btnPause">
           <property name="text">
            <string>Pause</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btnStop">
           <property name="text">
            <string>Stop</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="hspacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>0</width>
             <height>0</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QComboBox" name="cmbSpeed">
           <property name="toolTip">
            <string>Production speed</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>30</y>
          <width>630</width>
          <height>142</height>
         </rect>
        </property>
        <layout class="QGridLayout" name="stationsLayout">
         <item row="0" column="0">
          <widget class="QGroupBox" name="grpAssembler">
           <property name="title">
            <string>Assembler</string>
           </property>
           <layout class="QVBoxLayout">
            <item>
             <widget class="QLabel" name="lblAsmState">
              <property name="text">
               <string>Idle</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="lblAsmProcessed">
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QProgressBar" name="barAsm">
              <property name="value">
               <number>0</number>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QGroupBox" name="grpTester">
           <property name="title">
            <string>Tester</string>
           </property>
           <layout class="QVBoxLayout">
            <item>
             <widget class="QLabel" name="lblTesState">
              <property name="text">
               <string>Idle</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="lblTesProcessed">
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QProgressBar" name="barTes">
              <property name="value">
               <number>0</number>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item row="0" column="2">
          <widget class="QGroupBox" name="groupBoxQC">
           <property name="title">
            <string>Quality Control</string>
           </property>
           <widget class="QLabel" name="lblQCProcessed">
            <property name="geometry">
             <rect>
              <x>20</x>
              <y>70</y>
              <width>66</width>
              <height>18</height>
             </rect>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
           <widget class="QLabel" name="lblQCState">
            <property name="geometry">
             <rect>
              <x>10</x>
              <y>30</y>
              <width>66</width>
              <height>18</height>
             </rect>
            </property>
            <property name="text">
             <string>Q</string>
            </property>
           </widget>
           <widget class="QProgressBar" name="barQC">
            <property name="geometry">
             <rect>
              <x>10</x>
              <y>100</y>
              <width>118</width>
              <height>23</height>
             </rect>
            </property>
            <property name="value">
             <number>0</number>
            </property>
           </widget>
          </widget>
         </item>
         <item row="0" column="3">
          <widget class="QGroupBox" name="grpPacker">
           <property name="title">
            <string>Packer</string>
           </property>
           <layout class="QVBoxLayout">
            <item>
             <widget class="QLabel" name="lblPacState">
              <property name="enabled">
               <bool>true</bool>
              </property>
              <property name="text">
               <string>Idle</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="lblPacProcessed">
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QProgressBar" name="barPac">
              <property name="value">
               <number>0</number>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QPlainTextEdit" name="txtLogs">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>330</y>
          <width>630</width>
          <height>260</height>
         </rect>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
